rasa/train.py

```diff
@@ -134,10 +134,11 @@ def train(
     Returns:
         Path of the trained model archive.
     """
-    new_fingerprint = await model.model_fingerprint(file_importer)
-
-    stories = await file_importer.get_stories()
-    nlu_data = await file_importer.get_nlu_data()
```

Comment:
How was the fingerprint created before getting the stories and NLU before? Doesn't the fingerprint depend on the stories and such?

Reply:
just depends on the file importer. Moved it back down